libraries/botframework-schema/src/teams/index.ts

```diff
@@ -176,6 +184,11 @@ export interface TeamsChannelData {
      * message was sent
      */
     meeting?: TeamsMeetingInfo;
+    /**
+     * @member {TeamsChannelDataSettings} [settings] Information about the settings in which the
+     * message was sent
```

Comment:
```suggestion
     * message was sent.
```

Reply:
I left it without the dot because the rest doesn't have one, I could add the dot for the rest as well, so it stays consistent

Reply:
Ok, you can leave it without the dot, as you prefer.